### ISSUE TITLE
Python 3.8: module 'cgi' has no attribute 'escape'

### ISSUE THREAD

**Issue by fdelapena** [thp/urlwatch#477]
It seems `cgi.escape` was deprecated in 3.7 and removed 3.8. A possible alternative is `html.escape`:

```
Traceback (most recent call last):
  File "/usr/bin/urlwatch", line 111, in <module>
    urlwatch_command.run()
  File "/usr/lib/python3.8/site-packages/urlwatch/command.py", line 295, in run
    self.urlwatcher.close()
  File "/usr/lib/python3.8/site-packages/urlwatch/main.py", line 99, in close
    self.report.finish()
  File "/usr/lib/python3.8/site-packages/urlwatch/handler.py", line 159, in finish
    ReporterBase.submit_all(self, self.job_states, duration)
  File "/usr/lib/python3.8/site-packages/urlwatch/reporters.py", line 103, in submit_all
    subclass(report, cfg, job_states, duration).submit()
  File "/usr/lib/python3.8/site-packages/urlwatch/reporters.py", line 408, in submit
    body_html = '\n'.join(self.convert(HtmlReporter).submit())
  File "/usr/lib/python3.8/site-packages/urlwatch/reporters.py", line 149, in submit
    yield from (str(part) for part in self._parts())
  File "/usr/lib/python3.8/site-packages/urlwatch/reporters.py", line 149, in <genexpr>
    yield from (str(part) for part in self._parts())
  File "/usr/lib/python3.8/site-packages/urlwatch/reporters.py", line 183, in _parts
    yield SafeHtml(title).format(verb=job_state.verb,
  File "/usr/lib/python3.8/site-packages/urlwatch/reporters.py", line 144, in format
    **{k: cgi.escape(str(v)) for k, v in kwargs.items()})
  File "/usr/lib/python3.8/site-packages/urlwatch/reporters.py", line 144, in <dictcomp>
    **{k: cgi.escape(str(v)) for k, v in kwargs.items()})
AttributeError: module 'cgi' has no attribute 'escape'
```

Triggered with some rule like this:

```
kind: url
name: Example
url: https://example.org/
filter: xpath://*[@id="someid"]
```

Then update the document and run urlwatch, it will drop the traceback from above.

**Comment by cfbao** [comment]
This is handled in #424, though not released yet.

**Comment by thp** [comment]
Fixed in 2.18.
